invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/communities.js

```diff
@@ -49,28 +51,48 @@ function ResultsGridItemTemplate({ result, index }) {
 }
 
 export function CommunitiesResultsItemTemplate({ result, index }) {
+  const community_type = _get(
+    result,
+    "metadata.type",
+    i18next.t("No community type")
+  );
   return (
     <Item key={index}>
-      {index == 0 ? (
-        <Item.Image size="tiny" src={result.links.logo} />
-      ) : (
-        <Item.Image size="tiny" src="/static/images/placeholder.png" />
-      )}
+      <Item.Image size="tiny" src={result.links.logo} />
       <Item.Content>
+        <Item.Extra className="user-communities labels-actions">
+          {/* For reduced spacing between labels. */}
+          <span>
+            <Label size="tiny" color="grey">
+              {community_type}
+            </Label>
+          </span>
+          <Button
+            compact
+            size="small"
+            floated="right"
+            href={`/communities/${result.id}/settings`}
```

Comment:
I will create an issue for that, also because we need to solve how we can pass around with context these urls as these components are overriding already defined ones. WDYT?

Reply:
sure, an issue is a good idea

Reply:
https://github.com/inveniosoftware/invenio-app-rdm/issues/1211